### PR TITLE
results page

### PR DESCRIPTION
**Pull request by annaboomsma**
let me know what you guys think!!!

### CODE REVIEW

**Comment by annaboomsma** [comment]
oh btw the profile page is not completely done. Im still working on that

File: views/filter.ejs [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
+                    <input id="straight" class="radio" type="radio" name="sexuality" value="Straight">
+                    <label for="straight" class="label">Straight</label>
+                    <input id="gay" class="radio" type="radio" name="sexuality" value="Gay">
+                    <label for="gay" class="label">Gay</label>
+                    <input id="bi" class="radio" type="radio" name="sexuality" value="Bi-sexual">
+                    <label for="bi" class="label">Bi-sexual</label>
+                    
+                </fieldset>
+        </div>
+            <div class="submitButton">
```
Comment (partychickenking) [inline views/filter.ejs:39]: styling with 
input[type="submit"]{

}
Instead of using div

File: views/filter.ejs [hunk 2]
```diff
@@ -1,20 +1,44 @@
...
+
+<body>
+    <header>
+        <!-- <button id="backbutton" href="" ></button> -->
+        <img id="logo" src="../static/images/logo.png" alt="logo luvely">
+    
+    </header>
+    <main class="normalmain">
+        <form action="/" method="POST">
+        <div class="form">
```
Comment (partychickenking) [inline views/filter.ejs:17]: Using the styling on the form instead of the div

**Comment by tsjuusmei** [comment]
looks fine!

File: views/filter.ejs (hunk 1 above)
Comment (annaboomsma) [reply views/filter.ejs:39]: That div is only for styling

File: views/filter.ejs (hunk 2 above)
Comment (annaboomsma) [reply views/filter.ejs:17]: The div is only for styling

**Comment by annaboomsma** [comment]
I forgot to connect the database in a function so I'm going to change that and make a new pull request

**Pull request closed without merging**